### PR TITLE
Masaladosa

### PR DESCRIPTION
**Pull request by nallamsrikanth12**

### CODE REVIEW

**Comment by srikanthnallam** [comment]
why u add the extra karam and malasa dosa ?

**Comment by nallamsrikanth12** [comment]
> why u add the extra karam and malasa dosa ?

this ingredents add the dosa. dosa will  be more tasty

File: menu.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 dosa
-karam 
-oil
+extra karam 
+malasa dosa
```
Comment (srikanthnallam) [inline menu.txt:3]: any paritucular reason to add the masala ?

Comment (nallamsrikanth12) [reply menu.txt:3]: yes to become more tasty

**Review by srikanthnallam — COMMENTED** [review]
everything is good and go head

**Review by srikanthnallam — APPROVED** [review]

**Pull request merged**